cmd/event-notifier_test.go

```diff
@@ -93,7 +93,7 @@ func TestInitEventNotifier(t *testing.T) {
 	if err != nil {
 		t.Fatal("Unable to create directories for FS backend. ", err)
 	}
-	fs, err := getSingleNodeObjectLayer(disk[0])
+	fs, _, err := getSingleNodeObjectLayer(disk[0])
```

Comment:
All callers of getSingleNodeObjectLayer and getXLObjectLayer are ignoring the returned []StorageAPI. Is it worth returning it if no one requires it now?